content/en/docs/Responses/AlbumID3.md

```diff
@@ -59,12 +96,38 @@ description: >
 | `genre` | `string` | No |     | The album genre|
 | `played` | `string` | No | **Yes**    | Date the album was last played. [ISO 8601]|
 | `userRating` | `int` | No | **Yes**    | The user rating of the album. [1-5]|
+| `albumLabel` | `string` | No |  **Yes**   | The label producing the album. |
+| `musicBrainzId` | `string` | No |  **Yes**   | The album MusicBrainzID. |
+| `genres` | Array of `string` | No | **Yes**    | The list of all genres of the album. |
+| `artists` | Array of [`ArtistID3`](../artistid3) | No | **Yes**    | The list of all album artists of the album. (Note: Only the required [`ArtistID3`](../artistid3) fields should be returned by default)|
+| `displayArtist` | `string` | No |  **Yes**   | The single value display artist. |
+| `releaseType` | `string` | No | **Yes**    | The type of release. If the server handle subtypes then the field should return them type and subtypes separated with " / ".|
```

Comment:
Why not an array?
Should we try to normalize some possible values or at least give some examples?
Furthermore, I would expect to put here "Compilation" (like in https://musicbrainz.org/doc/Release_Group/Type)

Reply:
It was discussed already, but to resume no other server do support multi fields, and the tag is single valued and can contain anything.
There's no point in forcing an array here for very little value gain. (Including impacts on the servers to split the value + normalize and everything, when every other servers just expose the raw value).

Reply:
For me this is an important field, it allows to filter Singles, EPs, Demos, Lives, etc. easily in the artist view.
Having a "free" string will make for sure each client parse for some values for each case it wants to support, like "EP", "ExtendedPlay", "Extended play" etc. (ignoring case)
But I guess it is better than not field at all.

Reply:
In all cases single string or array, normalisation can't be enforced, user can put what they want in the tags.

From the logs I have from many users, there's tons of different values, and no easy way to normalize anything for those users.

In Symfonium I store all different values and exposes them to the smart filter stuff for easy selection, I do not care what the strings are and users are happy with that.